<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpaceSwitcherManager</class>
 <widget class="QWidget" name="SpaceSwitcherManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hl_selectedControl">
     <item>
      <widget class="QLabel" name="lbl_selectedControlsListLabel">
       <property name="text">
        <string>Selected Controls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="sa_selectedControlsList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sa_selectedControlsListContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>472</width>
        <height>68</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_selectedControlsList">
         <property name="text">
          <string>None</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_upperOptions">
     <item>
      <widget class="QCheckBox" name="cb_keyEnabled">
       <property name="text">
        <string>Key Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_forceKeyIfAlreadyAtValueEnabled">
       <property name="text">
        <string>Force Key if Already At Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_autoRefreshEnabled">
       <property name="text">
        <string>Auto Refresh Selections</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="sa_spacesList">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sa_spacesListContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>472</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
